pages/business-lines.md

```diff
@@ -3,4 +3,39 @@ permalink: /business-lines/
 title: Business lines
 ---
 
-18F has a variety of services grouped under three business lines, and is exploring additional services to offer. The links on the left go to pages with more information about each.
+18F has a variety of services grouped under five business lines, and is exploring additional services to offer. The links on the left go to pages with more information about each.
+
+##**Custom Partner Solutions (our first type of work):**
```

Comment:
Put a space after every `##`, change every `##` to `###`, and strip out all the `**`s.